CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Update opentelemetry-proto to v0.12.0. Note that this update removes deprecated status codes.
   ([#2415](https://github.com/open-telemetry/opentelemetry-python/pull/2415))
+- [exporter/opentelemetry-exporter-otlp-proto-grpc] Add Histogram to OTLPMetricExporter
```

Comment:
Are we adding metrics related changes to the changelog? I think we have been a little inconsistent on this.

Reply:
Yeah I'm not sure what makes sense here. I figured this would make sense if people wanted to know what types were supported in the otlp exporter 🤷

Reply:
Yeah maybe we should only put entries related to otlp? Whereas api/sdk we don't until we actually release? Not sure what to do here.

Reply:
I think we should not add a changelog entry for this change because these changes are in the still private module `.../_metric_exporter/__init__.py`.

Reply:
Are you suggesting we remove all entries for private modules then? If so we'd have to remove all mentions of metrics and logs as well.

Reply:
Well, at least I was not adding entries for changes in private modules, not sure if others think differently. :sweat_smile:

I am of this opinion because the changelog should not inform users of new features or bugfixes that are not yet supposed to be used by them.

Reply:
Maybe we should remove those entries? It would be more like a big consistency nit, I don't think it is critical.

Reply:
@ocelotl 
> I am of this opinion because the changelog should not inform users of new features or bugfixes that are not yet supposed to be used by them.

I agree with this point. My question is what would our changelog look like when we finally do release metrics and logs? Simply, "released metrics module" for example? Would there be a related PR that's linked? Also I don't want to block this PR for this question so maybe we can just remove the changelog entry for this PR for now and bring up this topic in the SIG? @codeboten thoughts?

Reply:
Sure that's fine with me.